src/lib/learn-client/api/collection/fetch-product-collections.ts

```diff
@@ -11,11 +16,26 @@ export const PRODUCT_COLLECTION_API_ROUTE = (
  * Collections will be product associated if they have at least
  * one tutorial whose primary product (first in frontmatter array)
  * matches this product slug
+ *
+ * Optional query params will return collections 
+ * sorted according to the category sidebar sort (which automatically
+ * includes filtering for theme
  */
 export async function fetchAllCollectionsByProduct(
-  identifier: ProductOption | uuid
+  product: AllCollectionsProductOptions
 ): Promise<Collection[]> {
-  const route = PRODUCT_COLLECTION_API_ROUTE(identifier)
+  const baseUrl = PRODUCT_COLLECTION_API_ROUTE(product.slug)
+  let route = baseUrl
+
+  if (product.sidebarSort) {
+    const params = new URLSearchParams([
+      ['topLevelCategorySort', 'true'],
+      ['theme', product.slug],
```

Comment:
Turns out that the `theme` filtering _only_ works in combination with sidebar sort. Made a ticket to follow up in the API board but not a blocker for now, we can filter on the frontend.